gpus/partials/_ollama-create.erb

```diff
@@ -0,0 +1,49 @@
+Running `ollama` on a GPU is pretty easy. The fastest way to get started is to adapt the following `fly.toml` file:
```

Comment:
**For all the partials:** We want to move all the partials under the main `partials` directory, with sub-directories there, so that all the partials are easy to find. For this PR, could you move any new ones you're creating? (No worries about existing partials, I'll fix those later!)
partials/tigris/
partials/postgres/
partials/gpus/